sources/packages/web/src/helpers/helpers/uri.ts

```diff
@@ -61,6 +62,9 @@ export const getPaginationQueryString = (
 ) => {
   if (enableZeroPage) {
     paginationOptions.page = paginationOptions.page - 1;
+    if (paginationOptions.searchCriteria) {
```

Comment:
I see the point here but does it work when there is a filter that results in multiple pages? Can the user still navigate across the pages with a filter applied?

Reply:
nice catch

Reply:
I small ticket would be better @ann-aot. Do you mind creating one?